installer/inventory

```diff
@@ -26,14 +26,18 @@ awx_secret_key=awxsecret
 # openshift_user=developer
 # awx_node_port=30083
 
-# Standalone Docker Install
+# Docker Compose Install
+use_docker_compose=false
```

Comment:
ideally these would be commented out and we'd use the `|default` semantic in the playbooks.

Also, put these options below the other docker parameters.

Reply:
Ok for commenting and using `default`
What's the rationale for the order ? Given that it's one of the first choices being given in the Install documentation, I would have said it makes sense to display the choice proheminently ...? (I'm not denying the change, I'm just looking at a better understanding for its reason, so I might do it better next time :) )